### DISCUSSION TITLE
Using nix — github:eza-community/eza#defaultApp.x86_64-linux

### DISCUSSION THREAD

**Discussion by ghost** [eza-community/eza#658] [Q&A, Howto, Help] [answered]
I wanted to give nix a try, I think I've installed it properly. But if I do

```
nix run github:eza-community/eza
```

(the command given in `README.md`) I get

```
error: cannot find flake attribute 'github:eza-community/eza#defaultApp.x86_64-linux'
```

So is there something wrong with my nix install, or just something else I need to do?

**Comment by cafkafk** [comment, marked as answer]
Hmm, it does work for me. I wonder what the output of `cat /etc/nix/nix.conf` is for you, if you have that file (not sure if it's NixOS only), as well as the output of `nix --version`. 

Do you have the experimental features `nix-command flakes repl-flake` enabled?
